### ISSUE TITLE
Add SDK version / level as define to a header

### ISSUE THREAD

**Issue by dueringa** [saleae/AnalyzerSDK#5]
There are differences between the SDK version 1.1.8 (1.1.14) and the current one.
The support pages (http://support.saleae.com/hc/en-us/articles/208667936) state that you must implement your analyzer differently.

Especially switching back between old and new SDK (for debugging) would be much easier if the SDK version number would be supplied in some header file, so you could simply use `#ifdef`s and react to whatever SDK directory you set your include path to.

It might not be possible to add this to old SDK releases (they would have to be re-released?), but the current SDK could maybe add this, or alternatively a `#define SDK_LEVEL 2` (or similar).

**Comment by Marcus10110** [comment]
Definitely. This is planned, as well as a few other improvements that will let the master branch build for both the latest SDK and the legacy SDK on Windows. There is a lot of stuff to do here, including getting all of Saleae's analyzers on github too, but it's going to be a while before I can get back to it, there is another project that's going to hold our attention for a while.
